runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -975,6 +975,15 @@ class CompilationInfo
    static void storeAOTInSharedCache(J9VMThread *vmThread, J9ROMMethod *romMethod, const U_8 *dataStart, UDATA dataSize, const U_8 *codeStart, UDATA codeSize, TR::Compilation *comp, J9JITConfig *jitConfig, TR_MethodToBeCompiled *entry);
 #endif
    static bool canRelocateMethod(TR::Compilation * comp);
+#if defined(JITSERVER_SUPPORT)
```

Comment:
In jitaas branch we must have this ifdef enabled by default. Is that the case?

Reply:
Yes, it's already enabled in `runtime/compiler/compiler.mk` 
```
ifeq ($(PLATFORM),amd64-linux64-gcc)
    export JITSERVER_SUPPORT=1
endif
```